lettuce/cli.py

```diff
@@ -60,10 +63,16 @@ def benchmark(ctx, steps, resolution, profile_out):
         lattice = LatticeAoS(D2Q9, device, dtype)
     else:
         lattice = Lattice(D2Q9, device, dtype)
-    flow = TaylorGreenVortex2D(resolution=resolution, reynolds_number=1, mach_number=0.05, lattice=lattice)
-    collision = BGKCollision(lattice, tau=flow.units.relaxation_parameter_lu)
+
+    flow_class = flow_by_name(flow)
+    flow = flow_class(resolution=resolution, reynolds_number=1, mach_number=0.05, lattice=lattice)
+    force = Guo(lattice, tau=0.6, F=flow.F)
+    collision = BGKCollision(lattice, tau=0.6, force=force)
```

Comment:
Is there a way to encapsulate this logic into the simulation class somehow? I need to think about this a little and get back to you later.

What is wrong with `tau=flow.units.relaxation_parameter_lu`?

Reply:
1. Can you add a force conversion to the units module and call it here?
(you can use the velocity conversion (back and forth) as a prototype and derive the right scaling factors from density, velocity, and length by dimension analysis)
2. Just add the force if flow has a force 


```
if hasattr(flow, "F"):
    force = Guo(lattice, tau=0.6, F=flow.units.convert_force_to_lu(flow.F))
else:
    force = None
```

That should be sufficient for now. At some point we can think about moving this to the simulation class and do something like `collision.force = ...` but let's keep it here for now.